src/components/validations/validations.stories.js

```diff
@@ -179,4 +182,39 @@ storiesOf('Validations', module)
         </Row>
       </Form>
     );
+  })
+  .add('ButtonToggleGroup', () => {
+    const test = value => new Promise((resolve, reject) => {
+      if (value === 'baz') return resolve(true);
+      return reject(Error('Baz is required!'));
+    });
+
+    return (
+      <Form
+        onSubmit={ (ev) => { ev.preventDefault(); } }
+      >
+        <State store={ buttonToggleGroupStore }>
+          <ButtonToggleGroup
+            label='ButtonToggle Validation'
+            labelHelp='Returns error unless "Baz" value selected'
+            fieldHelp='Click save to run validation'
+            validations={ test }
+          >
+            {['foo', 'bar', 'baz'].map(value => (
+              <ButtonToggle
+                name='button-toggle-group'
+                id={ value }
+                value={ value }
+                onChange={ (ev) => {
+                  buttonToggleGroupStore.set({ value: ev.target.value });
```

Comment:
Is it possible to have actions logs here?

Reply:
added